css/home.css

```diff
@@ -3,12 +3,16 @@ html, body {
 }
 
 .home_head_bar {
-    background: -webkit-linear-gradient(top, #373737, #212121);
+    background-color: #474747; /*fallback for browser does not support gradient*/
+    background: -webkit-linear-gradient(top, #373737, #212121);／＊Each of the most popular browser rendering engines has its own vendor-specific prefix for experimental properties.*/
```

Comment:
the comments should be more specific, like -o-linear-gradient is for Opera 11.1 to 12.0, we should put comment for each of these variants